reframe/frontend/cli.py

```diff
@@ -595,28 +595,33 @@ def print_infoline(param, value):
 
         options.flex_alloc_nodes = options.flex_alloc_nodes or 'idle'
         if options.account:
+            options.job_options.append(f'account={options.account}')
             printer.warning(f"`--account' is deprecated and "
                             f"will be removed in the future; you should "
                             f"use `-J account={options.account}'")
 
         if options.partition:
+            options.job_options.append(f'partition={options.partition}')
```

Comment:
@vkarak do we want to keep the option working only for slurm? Is there a way to pass this argument for both pbs and slurm through `job_options`?